lib/librte_hash/rte_cuckoo_hash.c

```diff
@@ -2659,3 +2659,32 @@ rte_hash_iterate(const struct rte_hash *h, const void **key, void **data, uint32
 	(*next)++;
 	return position - 1;
 }
+
+void
+rte_hash_prefetch_buckets(const struct rte_hash *h, hash_sig_t sig)
+{
+	uint16_t short_sig = get_short_sig(sig);
+	uint32_t prim_index = get_prim_bucket_index(h, sig);
+	uint32_t sec_index = get_alt_bucket_index(h, prim_index, short_sig);
+
+	const struct rte_hash_bucket *primary_bkt = &h->buckets[prim_index];
+	const struct rte_hash_bucket *secondary_bkt = &h->buckets[sec_index];
```

Comment:
Drop variables `primary_bkt` and `secondary_bkt` and call `rte_prefetch0()` directly. Do the same for `rte_hash_prefetch_buckets_non_temporal()`.